cloudferrylib/base/image.py

```diff
@@ -39,20 +39,21 @@ def glance_image_download_cmd(config, image_id, destination_file):
         file=destination_file)
 
 
-def glance_image_create_cmd(config, image_name, disk_format, filename,
+def glance_image_create_cmd(config, image_name, disk_format, filepath,
                             container_format="bare"):
     """Generates glance command which creates image based on arguments
     provided. Command output is filtered for 'id'
 
     :returns: Openstack CLI command"""
-    args = ("--name {image_name} "
+    args = ("image-create "
+            "--name {image_name} "
             "--disk-format={disk_format} "
             "--container-format={container_format} "
-            "--file {file}").format(
+            "--location {loc}").format(
```

Comment:
Unfortunately there are other places which require `--file` argument to be passed:
 - `cloudferrylib.os.actions.upload_file_to_image.UploadFileToImage`
 - `cloudferrylib.os.image.glance_image.GlanceImage#glance_img_create`
Please rewrite to make sure those keep the same logic.